src/helpers.ts

```diff
@@ -121,13 +121,13 @@ export function cssPath(): string {
       !node.getAttribute('id') &&
       !node.getAttribute('class')
     ) {
-      result += `[type="${node.getAttribute('type')}"]`;
+      result += `[type="${CSS.escape(node.getAttribute('type'))}"]`;
```

Comment:
You could drop the `"`:

```suggestion
      result += `[type=${CSS.escape(node.getAttribute('type'))}]`;
```

`CSS.escape` guarantees valid CSS identifier output, so no need to wrap it in a string literal — unless you want to.

Reply:
Ah, thanks!